Firmware/Marlin_main.cpp

```diff
@@ -6404,8 +6404,8 @@ SERIAL_PROTOCOLPGM("\n\n");
 #endif
         {
             const char* _msg_mode_done_remain = _N("%S MODE: Percent done: %d; print time remaining in mins: %d; Change in mins: %d\n");
-            printf_P(_msg_mode_done_remain, _N("NORMAL"), int(print_percent_done_normal), print_time_remaining_normal, print_time_to_change_normal);
-            printf_P(_msg_mode_done_remain, _N("SILENT"), int(print_percent_done_silent), print_time_remaining_silent, print_time_to_change_silent);
+            printf_P(_msg_mode_done_remain, _N("NORMAL"), int8_t(print_percent_done_normal), print_time_remaining_normal, print_time_to_change_normal);
```

Comment:
%hhd instead of %d is probably the most correct way of printing a signed int8_t

should be supported in gcc 4.9.x that we use